CMakeLists.txt

```
@@ -0,0 +1,47 @@
+# Copyright (c) Microsoft Corporation
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required(VERSION 3.22)
```

Is 3.22 really required?   I seem to only have 3.20 installed as part of VS 2019 and the GettingStarted.md currently doesn't say anything about a newer version of cmake needed.

---

I tried it on my machine after updating this line to 3.20 and it compiled successfully.
```suggestion
cmake_minimum_required(VERSION 3.20)
```

---

The failures I am experiencing are actually two: 
 * **bpftool**: This is failing locally on my env. It is currently marked as EXCLUDE_FROM_ALL, so it needs to be built explicitly with `cmake --build build --config Debug --target bpftool`
 * **execution_context_{user,kernel}**: Compiling fine on my env, by failing on the CI